doc/pages/source/documentation/index.md

```diff
@@ -950,7 +950,24 @@ to the chain of middleware execution.
 ### Sub-plans
 
 -   *when to use?*
--   *how to use?*
+
+To use sub-plans, you must include the `Dynflow::Action::WithSubPlans` module
+and override the `create_sub_plans` method. Inside the `create_sub_plans`
+method, you use the `trigger` method to create sub-tasks that will be executed
+in sequence but will not block the parent task.
+
+```rb
+class MyAction < Actions::EntryAction
+  include Dynflow::Action::WithSubPlans
+
+  ...
+
+  def create_sub_plans
+    trigger(Actions::OtherAction, action_param1, action_opts)
+    trigger(Actions::AnotherAction)
```

Comment:
The parent task expects this method to return either a single task (in case it has only one sub-task) or an array of them (in case of multiple sub-tasks). In this example the parent task would lose track of `OtherAction` because it is not returned from the method.